### PR TITLE
Css Rework & Survivalist Tip

### PR DESCRIPTION
**Pull request by IDKFORCE**

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -82,7 +50,7 @@ end)
 exports['qbr-core']:CreateCallback("qb-multicharacter:server:setupCharacters", function(source, cb)
     local license = exports['qbr-core']:GetIdentifier(source, 'license')
     local plyChars = {}
-    MySQL.query('SELECT * FROM players WHERE license = @license', {['@license'] = license}, function(result)
+    MySQL.Async.fetchAll('SELECT * FROM players WHERE license = @license', {['@license'] = license}, function(result)
```
Comment (gononono64) [inline server/main.lua:53]: This does not follow oxmysql

File: server/main.lua [hunk 2]
```diff
@@ -112,7 +80,7 @@ exports['qbr-core']:CreateCallback("qb-multicharacter:server:GetNumberOfCharacte
 end)
 
 exports['qbr-core']:CreateCallback("qbr-multicharacter:server:getSkin", function(source, cb, cid)
-    MySQL.query('SELECT * FROM playerskins WHERE citizenid = ? AND active = ?', {cid, 1}, function(result)
+    MySQL.Async.fetchAll('SELECT * FROM playerskins WHERE citizenid = ? AND active = ?', {cid, 1}, function(result)
```
Comment (gononono64) [inline server/main.lua:83]: This does not follow oxmysql

**Review by gononono64 — CHANGES_REQUESTED** [review]
MySQL.Async.fetchAll needs to be replaced with MySQL.query

**Review by gononono64 — CHANGES_REQUESTED** [review]
MySQL.Async.fetchAll needs to be replaced with MySQL.query

**Comment by IDKFORCE** [comment]
good looks i forgot about that

**Pull request closed without merging**